tests/testthat/test-vivax.R

```diff
@@ -0,0 +1,26 @@
+test_that('Test falciparum switch produces same', {
+  parameters_def <- get_parameters()
+  parameters_fal <- get_parameters(parasite = "falciparum")
+  expect_identical(parameters_def, parameters_fal)
+})
+
+test_that('Test vivax model runs', {
+  vivax_parameters <- get_parameters(parasite = "vivax")
+  sim_res <- run_simulation(timesteps = 100, parameters = vivax_parameters)
+  expect_equal(nrow(sim_res), 100)
+})
+
+test_that('Test parasite = vivax sets parameters$parasite to vivax', {
+  vivax_parameters <- get_parameters(parasite = "vivax")
+  expect_identical(vivax_parameters$parasite, "vivax")
+})
+
+test_that('Test difference between falciparum and vivax parameter lists', {
+  falciparum_parameters <- get_parameters(parasite = "falciparum")
+  vivax_parameters <- get_parameters(parasite = "vivax")
+  
+  expect_true(all(names(falciparum_parameters)[!names(falciparum_parameters) %in% names(vivax_parameters)] %in%
```

Comment:
This I suspect isn't doing what you expect. 
`names(falciparum_parameters)[!names(falciparum_parameters) %in% names(vivax_parameters)]` is `character(0)`.
I'm not 100% sure what you're testing, so more explicit steps would help. I've re-written some as an example here:

```
test_that('Test difference between falciparum and vivax parameter lists', {
  falciparum_parameters <- get_parameters(parasite = "falciparum")
  vivax_parameters <- get_parameters(parasite = "vivax")
  
  vivax_specific_parameters <- c(
    "du","rvm","rva","rb","b0","b1","ib0","kb","theta0","theta1","kv","fv0",
    "av","gammav","iv0","fd0","ad","gammad","d1","id0","kd","ub","uv","gamma1",
    "pvm","init_ivm","init_ib","init_iva"
  )
  
  in_vivax_not_falciparum <- setdiff(names(vivax_parameters), names(falciparum_parameters))
  expect_identical(
    sort(in_vivax_not_falciparum),
    c("alm50", "apcr50", "b", "ca", "dpcr_max", "dpcr_min", "f", 
      "gammal", "init_hyp", "init_iaa", "init_iam", "klm", "kmax", 
      "kpcr", "philm_max", "philm_min", "ra", "ua")
  )
})
```

Reply:
Might also be worth thinking about how this would deal with future `Pf` parameter changes/additions - it might be quite unexpected for the dev to have to change `test-vivax.R` if making a `Pf` parameter name change for example?

Reply:
I've addressed the tests as we discussed.